src/main/java/com/example/NYTAPI/Models/Article.java

```diff
@@ -20,4 +20,19 @@ public class Article {
 
     private List<Media> media;
     private String imageUrl;
+
+    public Article(Long id, String url, String section, String subsection, String byline, String title, String summary, List<Media> media) {
+        this.id = id;
+        this.url = url;
+        this.section = section;
+        this.subsection = subsection;
+        this.byline = byline;
+        this.title = title;
+        this.summary = summary;
+        this.media = media;
+        if (!media.isEmpty()) {
+            this.imageUrl = getMedia().get(0).getMediaMetadata().get(1).getUrl();
```

Comment:
The `.get(<index>)` method assumes that:
- The list it is called on is not `null` 
- that the list is not empty
if any of theis is not true - it will result in `NullPointerError`. 
So it is a good thing to check the criterias before access.

Reply:
Ok, thanks! Submitting a new PR on this.